### PR TITLE
PXP-660 feat: add cloudsql database metrics panel

### PR DESCRIPTION
**Pull request by Fadhil**
## Summary

In this PR we introduce a third panel called "Databases" that pulls metrics from GCloud using the protobuf generated modules by tonic for gcloud.
Screenshot 2024-03-06 at 4 30 57 PM

Ticket: https://mindvalley.atlassian.net/browse/PXP-660

## What's Changed

* Network dispatching of new get_database_metrics mechanism.

* Changed `Block::Log` to `Block::Middle`

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/202#" title="Author self-approved">Fadhil</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/202#" title="Author self-approved">Fadhil</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/202#pullrequestreview-1921354185" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
